packages/manager/src/queries/cloudpulse/queries.ts

```diff
@@ -84,6 +84,6 @@ export const queryFactory = createQueryKeys(key, {
 
   token: (serviceType: string | undefined, request: JWETokenPayLoad) => ({
     queryFn: () => getJWEToken(request, serviceType!),
-    queryKey: [serviceType],
+    queryKey: [serviceType, { resource_ids: request.resource_ids.sort() }],
```

Comment:
Is the sort necessary? 

Could this be 
```suggestion
    queryKey: [serviceType, request],
```

or 

```suggestion
    queryKey: [serviceType, { resource_ids: request.resource_ids }],
```

Reply:
Sorting is necessary for preventing unnecessary token creation for differently ordered same resource ids..

Reply:
Ah, I understand. Thanks 👍